docs/user/dql/functions.rst

```diff
@@ -1448,6 +1450,47 @@ Example::
     | 4                               |
     +---------------------------------+
 
+    os> SELECT DAYOFWEEK(DATETIME('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +----------------------------------------------+
+    | DAYOFWEEK(DATETIME('2020-08-26 00:00:00'))   |
+    |----------------------------------------------|
+    | 4                                            |
+    +----------------------------------------------+
+
+    os> SELECT DAYOFWEEK(TIMESTAMP('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +-----------------------------------------------+
+    | DAYOFWEEK(TIMESTAMP('2020-08-26 00:00:00'))   |
+    |-----------------------------------------------|
+    | 4                                             |
+    +-----------------------------------------------+
+
+    os> SELECT DAY_OF_WEEK(DATE('2020-08-26'))
+    fetched rows / total rows = 1/1
+    +-----------------------------------+
+    | DAY_OF_WEEK(DATE('2020-08-26'))   |
+    |-----------------------------------|
+    | 4                                 |
+    +-----------------------------------+
+
+    os> SELECT DAY_OF_WEEK(DATETIME('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +------------------------------------------------+
+    | DAY_OF_WEEK(DATETIME('2020-08-26 00:00:00'))   |
+    |------------------------------------------------|
+    | 4                                              |
+    +------------------------------------------------+
+
+    os> SELECT DAY_OF_WEEK(TIMESTAMP('2020-08-26 00:00:00'))
+    fetched rows / total rows = 1/1
+    +-------------------------------------------------+
+    | DAY_OF_WEEK(TIMESTAMP('2020-08-26 00:00:00'))   |
+    |-------------------------------------------------|
+    | 4                                               |
+    +-------------------------------------------------+
```

Comment:
I'm curious, does `SELECT DAY_OF_WEEK("2020-08-26");` work and if so it should probably have a test.

Reply:
Thanks!